sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -120,7 +120,7 @@ def get_iter(state, unused_arg):
 
 def symmetric_binary_op(state, unused_arg):
   # TODO(robertwb): This may not be entirely correct...
-  b, a = state.stack.pop(), state.stack.pop()
+  b, a = Const.unwrap(state.stack.pop()), Const.unwrap(state.stack.pop())
```

Comment:
It looks like this is an unrelated change that got pulled in?

Reply:
This was needed for my test. (It now correctly infers `x + 1` for ints `x`.) I thought it small enough to not merit a new PR.